.github/workflows/export.yml

```diff
@@ -14,18 +14,17 @@ permissions:
   contents: write
 
 jobs:
+
   tests:
     uses: ./.github/workflows/full_tests.yml
-  export-sharly-chess:
+
+  export-sharly-chess-windows:
+    # Next job export-sharly-chess-windows needs to run on a Windows self-hosted GitHub runner
+    # for cloud code-signing. If the job doesn't start, it is probably because GitHub did not
+    # find a self-hosted runner with label 'your-github-id'.
+    # In this case, check the labels of your runners.
     needs: tests
-    strategy:
-      matrix:
-        include:
-          - os: windows-latest
-            platform: windows
-          - os: macos-latest
-            platform: macos
-    runs-on: ${{ matrix.os }}
+    runs-on: ${{ github.actor }}
```

Comment:
This is currently tied to your username, should we add other labels to the runner, so that Tim and I would be able to create a release?